site/app/views/submission/HomeworkView.php

```diff
@@ -505,7 +505,9 @@ public function getRandomIndices(int $array_length, string $student_id, string $
             'allowed_minutes' => $gradeable->getUserAllowedTime($this->core->getUser()),
             'can_student_submit' => $canStudentSubmit,
             'is_grader_view' => false,
-            'recent_version_url' => $recent_version_url
+            'recent_version_url' => $recent_version_url,
+            'git_auth_token_url' => $this->core->buildUrl(['git_auth_tokens']),
+            'git_auth_token_required' => false
```

Comment:
This (referring to git_auth_token_required) along with 49 on TeamView.php would be eventually replaced with something like:

`$this->core->getAuthentication() instanceof SamlAuthentication`

Right now there won't be any requirement to use these but with SAML auth these will be necessary.